tests/Optimizer2/Inspection/Inspection.cpp

```diff
@@ -23,28 +23,34 @@
 #include "gtest/gtest.h"
 
 #include "Basics/VelocyPackStringLiteral.h"
-#include <Inspection/VPackInspection.h>
-#include <Inspection/StatusT.h>
+#include <Inspection/VPackWithErrorT.h>
+// #include <Inspection/StatusT.h>
+
+// Mock includes
+#include "MockQuery.h"  // might not be needed
+#include "Transaction/StandaloneContext.h"
+#include "../../Graph/GraphTestTools.h"
+#include "Servers.h"
```

Comment:
this needs cleanup - currently not used as I've had to write the tests I wanted to in "SingleServerProviderTest.cpp"